packages/web/src/common/store/upload/sagas.js

```diff
@@ -339,7 +339,7 @@ function* uploadWorker(requestChan, respChan, progressChan) {
           ? makeConfirmerSuccessForCollection
           : makeConfirmerSuccess)(id, index),
         isCollection ? makeConfirmerFailureCollection(id) : confirmerFailure,
-        () => {},
+        () => { },
```

Comment:
will this fail lint, or was it fixed for lint?

Reply:
just auto changed when i saved